cli/src/main/java/uk/ac/ebi/atlas/cli/coexpression/CoexpressionUpdateCommand.java

```diff
@@ -0,0 +1,63 @@
+package uk.ac.ebi.atlas.cli.coexpression;
+
+import com.google.gson.JsonPrimitive;
+import org.springframework.stereotype.Component;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import uk.ac.ebi.atlas.experimentimport.coexpression.BaselineCoexpressionProfileLoader;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.logging.Logger;
+
+@Component
+@Command(
+        name = "update-baseline-coexpression",
+        description = "Update coexpression for baseline experiments")
+public class CoexpressionUpdateCommand implements Callable<Integer> {
+
+    private static final Logger LOGGER = Logger.getLogger(CoexpressionUpdateCommand.class.getName());
+
+    @Option(names = {"-e", "--experiment"}, split = ",", description = "one or more experiment accessions, comma separated", required = true)
+    private List<String> experimentAccessions;
+
+    private final BaselineCoexpressionProfileLoader baselineCoexpressionProfileLoader;
+
+    public CoexpressionUpdateCommand(BaselineCoexpressionProfileLoader baselineCoexpressionProfileLoader) {
+        this.baselineCoexpressionProfileLoader = baselineCoexpressionProfileLoader;
+    }
+
+    @Override
+    public Integer call() {
+        LOGGER.info("Starting update coexpression for accessions.");
+        int deleteCount;
+        int loadCount;
+
+        int done=0;
+        List<String> failed_accessions = new ArrayList<>();
```

Comment:
A wild snake-cased variable appears! ;)

Reply:
hehehe.... you don't imagine how many lines I wrote without `;`... this is python talking.